vuepress/docs/next/docs/getting-started/multitenancy.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Multitenancy
+
+## Overview
+
+Entando 7.2 introduces multitenancy to the Entando Platform. An Entando Application can be customized to enable a multitenant architecture where tenants share an infrastructure but are securely informationally isolated. This document provides an overview of multitenancy and its Entando implementation.
```

Comment:
Minor, "introduces" is language we'd use in the release notes. Maybe here that should just be a parenthetical note, e.g. (new with Entando 7.2)?

Reply:
@nshaw there are a number of instances in our documentation where we lead with or include the phrase "starting with Entando x.y"; i used that pattern here, but please let me know if parenthetical is preferable

Reply:
@Lyd1aCla1r3 let's keep it as-is. I don't feel strongly about it other than to check for this kind of thing when we release a new version.